content/en/blog/_posts/2022-XX-XX-iptables-chains.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: blog
+title: "Kubernetes’s IPTables Chains Are Not API"
+date: 2022-XX-XX
+slug: iptables-chains
+---
+
+### or, “Sorry, You Are Not Invited to SIG Network’s `MASQUERADE` Party”
+
+**Author:** Dan Winship (Red Hat)
+
+Some Kubernetes components (such as kubelet and kube-proxy) create
+IPTables chains and rules as part of their operation. These chains
+were never intended to be part of any Kubernetes API/ABI guarantees,
+but some external components nonetheless make use of some of them (in
+particular, using `KUBE-MARK-MASQ` to mark packets as needing to be
+masqueraded).
+
+In Kubernetes 1.25 we are declaring explicitly that (with one
+exception), the IPTables chains that Kubernetes creates are intended
+only for Kubernetes’s own internal use, and third-party components
+should not assume that Kubernetes will create any specific IPTables
+chains, or that those chains will contain any specific rules if they
+do exist.
+
+Then, in future releases, as part of [KEP-3178], we will begin phasing
+out certain chains that Kubernetes itself no longer needs. Components
+outside of Kubernetes itself that make use of `KUBE-MARK-MASQ`,
+`KUBE-MARK-DROP`, or other Kubernetes-generated IPTables chains should
+start migrating away from them now.
+
+[KEP-3178]: https://github.com/kubernetes/enhancements/issues/3178
+
+## Upcoming changes
+
+Starting a few releases from now, kubelet will no longer create the
```

Comment:
Can we be more specific about a minimum number of releases before this change happens?